cugraph-analytics/cugraph_analytics.ipynb

```diff
@@ -0,0 +1,453 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running Graph Analytics on Large Scale Graphs Effortlessly with Nvidia and Memgraph \n",
+    "\n",
+    "This tutorial will show you how to use  **PageRank** graph analysis and **Louvain** community detection on a **Facebook dataset** containing 1.3 million relationships. Upon completing it, you will know how to run analytics algorithms on your dataset using **Python**. Then, you can run any of the following algorithms:\n",
+    "* Balanced Cut (clustering)\n",
+    "* Spectral Clustering (clustering)\n",
+    "* HITS (hubs vs. authorities analytics)\n",
+    "* Leiden (community detection)\n",
+    "* Katz Centrality\n",
+    "* Betweenness Centrality\n",
+    "\n",
+    "All of the algorithms above are powered by **[Nvidia cuGraph](https://www.nvidia.com/en-us/)** and they will execute on **GPU**.\n",
```

Comment:
```suggestion
    "All of the algorithms above are powered by **[Nvidia cuGraph](https://www.nvidia.com/en-us/)** and they will execute on **GPU**.\n",
```
    "All of the algorithms above are powered by **[Nvidia cuGraph](https://rapids.ai/)** and they will execute on **GPU**.\n",